src/pudl/workspace/setup.py

```diff
@@ -58,139 +46,38 @@ def get_defaults(
         os.environ["PUDL_OUTPUT"] = str(Path("~/pudl-work/output").expanduser())
         os.environ["PUDL_INPUT"] = str(Path("~/pudl-work/data").expanduser())
 
-    yaml_settings = _munge_legacy_yaml(
-        yaml_file=yaml_file, default_pudl_yaml=default_pudl_yaml
-    )
+    if input_dir:
```

Comment:
Overriding the env vars here makes `input_dir` and `output_dir` arguments persist in subprocesses... that seems like less surprising behavior than before!